javatests/arcs/core/host/LifecycleTest.kt

```diff
@@ -212,4 +215,14 @@ class LifecycleTest {
         val particle: ReadWriteAccessParticle = testHost.getParticle(arc.id, name)
         assertThat(particle.errors).isEmpty()
     }
+
+    @Test
+    fun pipeline() = runTest {
+        val name = "PipelineConsumerParticle"
+        val arc = allocator.startArcForPlan(PipelineTestPlan).waitForStart()
+        val particle: PipelineConsumerParticle = testHost.getParticle(arc.id, name)
```

Comment:
This is a bit outside my expertise, but I want to double check: do we need an `idle()` here? For my own understanding, if we don't -- why not?

Reply:
Is this covered in `waitForStop()`?

Reply:
That's my understanding, yes.